rp235x-hal-examples/README.md

```diff
@@ -107,14 +220,13 @@ It is currently possible to build *some* of the examples in RISC-V mode. See
 work. The missing ones probably rely on interrupts, or some other thing we
 haven't ported to work in RISC-V mode yet.
 
+Here is an example using the `blinky` example in RISC-V mode:
 ```console
-$ cargo build --bin blinky --target=riscv32imac-unknown-none-elf
-    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.06s
-$ file ./target/riscv32imac-unknown-none-elf/debug/blinky
-./target/riscv32imac-unknown-none-elf/debug/blinky: ELF 32-bit LSB executable, UCB RISC-V, RVC, soft-float ABI, version 1 (SYSV), statically linked, with debug_info, not stripped
-$ cargo run --bin blinky --target=riscv32imac-unknown-none-elf
-    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.06s
-     Running `picotool load -u -v -x -t elf target/riscv32imac-unknown-none-elf/debug/blinky`
+$ cargo brr --bin blinky
+    Finished `release` profile [optimized] target(s) in 0.06s
+$ cargo rrr --bin blinky
```

Comment:
We already had examples of using aliases above.
I think keeping the non-aliased build commands here would make sense - keeps it obvious for folks not using these aliases.